share/picongpu/benchmarks/TWEAC-FOM/include/picongpu/param/fieldBackground.param

```diff
@@ -44,139 +48,52 @@ namespace picongpu
-                  false,
-                  /* Polarization of TWTS laser field */
-                  templates::twtsfast::EField::LINEAR_X)
-            , twtsFieldE3(
-                  /* focus_y [m], the distance to the laser focus in y-direction */
-                  30.0e-6,
-                  /* wavelength [m] */
-                  0.8e-6,
-                  /* pulselength [s], sigma of std. gauss for intensity (E^2) */
-                  10.0e-15 / 2.3548200450309493820231386529194,
-                  /* w_x [m], cylindrically focused spot size */
-                  1.2e-6,
-                  /* interaction angle between TWTS laser propagation vector and the y-axis [rad] */
-                  3.5 * (PI / 180.),
-                  /* propagation speed of overlap [speed of light]. */
-                  1.0,
-                  /* manual time delay [s] if auto_tdelay is false */
-                  50.0e-6 / sim.si.getSpeedOfLight(),
-                  /* Should PIConGPU automatically choose a suitable time delay? [true/false] */
-                  false,
-                  /* Polarization of TWTS laser field */
-                  templates::twtsfast::EField::LINEAR_YZ)
-            , twtsFieldE4(
-                  /* focus_y [m], the distance to the laser focus in y-direction */
-                  30.0e-6,
-                  /* wavelength [m] */
-                  0.8e-6,
-                  /* pulselength [s], sigma of std. gauss for intensity (E^2) */
-                  10.0e-15 / 2.3548200450309493820231386529194,
-                  /* w_x [m], cylindrically focused spot size */
-                  1.2e-6,
-                  /* interaction angle between TWTS laser propagation vector and the y-axis [rad] */
-                  -3.5 * (PI / 180.),
-                  /* propagation speed of overlap [speed of light]. */
-                  1.0,
-                  /* manual time delay [s] if auto_tdelay is false */
-                  50.0e-6 / sim.si.getSpeedOfLight(),
-                  /* Should PIConGPU automatically choose a suitable time delay? [true/false] */
-                  false,
-                  /* Polarization of TWTS laser field */
-                  templates::twtsfast::EField::LINEAR_YZ)
+            , twtsFieldEPos(
+                  Params::FOCUS_Y_SI,
+                  Params::WAVE_LENGTH_SI,
+                  Params::PULSE_LENGTH_SI,
+                  Params::W0_SI,
+                  Params::PHI,
+                  Params::BETA_0,
+                  Params::TDELAY,
+                  Params::AUTO_TDELAY,
+                  Params::Polarization)
+            , twtsFieldENeg(
+                  Params::FOCUS_Y_SI,
+                  Params::WAVE_LENGTH_SI,
+                  Params::PULSE_LENGTH_SI,
+                  Params::W0_SI,
+                  -Params::PHI,
+                  Params::BETA_0,
+                  Params::TDELAY,
+                  Params::AUTO_TDELAY,
+                  Params::Polarization)
```

Comment:
The sign of the polarization angle does not need to be flipped together with the sign of the propagation angle `phi`? (Same for B)

Reply:
Good catch.